### PR TITLE
Added 2 endpoints to retrieve filtered information for big order table

### PR DESCRIPTION
**Pull request by KizerovDmitriy**
# GreenCityUBS PR
## Summary Of Changes 🔥

- rollback [previus ](https://github.com/ita-social-projects/GreenCityUBS/pull/1469)changes.
- added 2 new endpoints '/city-list' and '/districts-list'.
- added new logic to retrieve cities and districts by region. 
- added tests.

## Issue link

[#7391](https://github.com/ita-social-projects/GreenCity/issues/7391)

# PR Checklist Forms

_(to be filled out by PR submitter)_
- [x] Code is up-to-date with the `dev` branch.
- [x] You've successfully built and run the tests locally.
- [x] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (ISSUE-XXXX: Ticket title)
- [x] This template filled (above this section).
- [x] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells or duplication
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

**Review by Maryna-511750 — APPROVED** [review]

File: core/src/test/java/greencity/controller/AdminUbsControllerTest.java [hunk 1]
```diff
@@ -149,4 +154,47 @@ void saveColumnWidthForEmployeeTest() throws Exception {
             .content(json))
             .andExpect(status().isOk());
     }
+
+    @Test
+    void getCitiesControllerTest() throws Exception {
+        List<UkraineRegion> regions = Arrays.asList(UkraineRegion.KYIV_OBLAST, UkraineRegion.LVIV_OBLAST);
+        List<CityDto> cities = Arrays.asList(new CityDto("Київ", "Kyiv"), new CityDto("Львів", "Lviv"));
+
+        when(ordersAdminsPageService.getAllCitiesByRegion(regions)).thenReturn(cities);
```
Comment (ABbondar) [inline core/src/test/java/greencity/controller/AdminUbsControllerTest.java:163]: please add verify

File: core/src/test/java/greencity/controller/AdminUbsControllerTest.java [hunk 2]
```diff
@@ -149,4 +154,47 @@ void saveColumnWidthForEmployeeTest() throws Exception {
...
+        assertEquals("Lviv", response.getBody().get(1).getCityEn());
+    }
+
+    @Test
+    void getDistrictsControllerTest() throws Exception {
+        String[] cities = {"Kyiv", "Lviv"};
+        List<DistrictDto> districts =
+            Arrays.asList(new DistrictDto("Район1", "District1"), new DistrictDto("Район2", "District2"));
+
+        when(ordersAdminsPageService.getAllDistrictsByCities(cities)).thenReturn(districts);
```
Comment (ABbondar) [inline core/src/test/java/greencity/controller/AdminUbsControllerTest.java:187]: please, add verify

File: service/src/test/java/greencity/service/ubs/OrdersAdminsPageServiceImplTest.java [hunk 3]
```diff
@@ -256,7 +256,7 @@ void getParametersForOrdersTest() {
             .thenReturn(addressList);
         when(addressRepository.findDistinctRegions())
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/ubs/OrdersAdminsPageServiceImplTest.java:257]: the same here

File: service/src/test/java/greencity/service/ubs/OrdersAdminsPageServiceImplTest.java [hunk 4]
```diff
@@ -1166,75 +1166,33 @@ void saveOrderColumnWidthForEmployeeIfNotPresentTest() {
...
-            .thenReturn(Optional.ofNullable(orderStatusTranslation2.setStatusId(5L)));
+    @Test
+    void testGetAllCitiesByRegion() {
+        List<UkraineRegion> regions = Arrays.asList(UkraineRegion.KYIV_OBLAST, UkraineRegion.LVIV_OBLAST);
+        List<String> regionNamesList = Arrays.asList("Kyivs'ka oblast", "Lvivs'ka Oblast");
+        List<CityDto> cities = Arrays.asList(new CityDto("Київ", "Kyiv"), new CityDto("Львів", "Lviv"));
 
-        when(orderStatusTranslationRepository.getOrderStatusTranslationById(6L))
-            .thenReturn(Optional.ofNullable(orderStatusTranslation.setStatusId(6L)));
+        when(addressRepository.findAllCitiesByRegions(regionNamesList)).thenReturn(cities);
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/ubs/OrdersAdminsPageServiceImplTest.java:1175]: the same here

**Review by lukovskyj — APPROVED** [review]

**Review by ABbondar — APPROVED** [review]

**Pull request merged**